src/components/MaxHeapVisualizer.vue

```diff
@@ -91,17 +132,31 @@ export default {
       newSequence.addFrame(frames.frames[0]);
       this.sequencesList.push(newSequence);
       this.current = this.sequencesList.length - 1;
+      this.currentFrame = 0;
       for (let i = 1; i < frames.frames.length; i += 1) {
         setTimeout(() => {
           newSequence.addFrame(frames.frames[i]);
           if (i === frames.frames.length - 1) {
             this.isDisabled = false;
           }
+          this.currentFrame += 1;
           this.sequencesList = Object.assign(this.sequencesList);
         }, i * 1000);
       }
     },
+    changeCurrentFrame(event) {
+      this.currentFrame += event;
+      if (this.currentFrame < 0) {
+        this.changeCurrent(-1);
+        this.currentFrame = this.sequencesList[this.current].frames.length - 1;
+      }
+      if (this.currentFrame > this.sequencesList[this.current].frames.length - 1) {
+        this.changeCurrent(1);
+        this.currentFrame = 0;
+      }
+    },
     changeCurrent(event) {
+      this.currentFrame = 0;
```

Comment:
In my opinion, after changing the sequence, it should start from the last frame not from the first.

Reply:
will do